internal/install/command.go

```diff
@@ -126,19 +128,31 @@ func validateProfile(maxTimeoutSeconds int, sg *segment.Segment) *types.DetailEr
 
 	if accountID == 0 {
 		errorOccured = true
-		detailErr = types.NewDetailError(types.EventTypes.AccountIDMissing, "account ID is required")
+		detailErr = types.NewDetailError(types.EventTypes.AccountIDMissing, "Account ID is required.")
 		return detailErr
 	}
 
 	if APIKey == "" {
 		errorOccured = true
-		detailErr = types.NewDetailError(types.EventTypes.APIKeyMissing, "API key is required")
+		detailErr = types.NewDetailError(types.EventTypes.APIKeyMissing, "User API key is required.")
+		return detailErr
+	}
+
+	if !utils.IsValidUserAPIKeyFormat(APIKey) {
+		errorOccured = true
+		detailErr = types.NewDetailError(types.EventTypes.InvalidUserAPIKeyFormat, `Invalid user API key format detected. Please provide a valid user API key. User API keys usually have a prefix of "NRAK-" or "NRAA-".`)
 		return detailErr
 	}
 
 	if region == "" {
 		errorOccured = true
-		detailErr = types.NewDetailError(types.EventTypes.RegionMissing, "region is required")
+		detailErr = types.NewDetailError(types.EventTypes.RegionMissing, "Region is required.")
```

Comment:
@sanderblue is region really required, or do we default to US.

Reply:
We default to the US in the client.